AspNetCore.CacheOutput/CacheOutputAttribute.cs

```diff
@@ -76,8 +76,14 @@ public int SharedTimeSpan
 
         /// <summary>
         /// Corresponds to CacheControl Private HTTP header. Response can be cached by browser but not by intermediary cache.
+        /// NOTE: This is defaulted to true for backwards compatibility (it was always locked to private previously) 
         /// </summary>
-        public bool Private { get; set; }
+        public bool Private { get; set; } = true;
+        
+        /// <summary>
+        /// CacheControl explicit public 
+        /// </summary>
+        public bool Public { get; set; }
```

Comment:
Of course, to use this, you now need to provide:

```
Private = false, Public = true
```

I was debating making `Public = !Private` when building a header, but that also feels wrong.

I guess another idea is to make Private and Public nullable. If Public and Private are null - we can set Private in the header (backwards compatibility), then if Public = true, set public header etc.